code/game/objects/effects/spiders.dm

```diff
@@ -67,7 +67,7 @@
 /obj/effect/spider/stickyweb/CanPass(atom/movable/mover, turf/target, height=0, air_group=0)
 	if(air_group || height == 0)
 		return TRUE
-	if(istype(mover, /mob/living/simple_animal/hostile/giant_spider) || istype(mover,/mob/living/simple_animal/hostile/spider_queen))
+	if(istype(mover, /mob/living/simple_animal/hostile/giant_spider) || istype(mover,/mob/living/simple_animal/hostile/giant_spider/nurse/spider_queen))
```

Comment:
you don't need the second check here anymore, spider_queen is now a subtype of the check before

Reply:
Handled